flambe/field/text.py

```diff
@@ -250,3 +239,45 @@ def process(self, example: str) -> torch.Tensor:  # type: ignore
             else:
                 ret = ret[:self.max_seq_len]
         return ret
+
+    def _get_embeddings(
```

Comment:
@brianasapp this doesn't need to be an instance method. I think it would be better if we add it as a module method (passing the appropriate methods). Splitting this unnecessary coupling will make testing easier (avoiding patching for example)

Reply:
And with the benefit of having this method available for other classes/methods in the codebase.

Reply:
Sure I agree, will separate it out